config/client.yaml

```diff
@@ -3,6 +3,9 @@ config:
     release-v1.15:
       konflux:
         enabled: true
+      imageOverrides:
+      - name: CLI_ARTIFACTS
+        pullSpec: brew.registry.redhat.io/rh-osbs/openshift-serverless-1-kn-cli-artifacts-rhel8:1.15.0
```

Comment:
@mgencur do you remember the list of allowed registries for Konflux?

Reply:
My understanding was going through ARGS and override it on the pipeline is preferred approach per Martin's suggestion here: https://github.com/openshift-knative/client/pull/525#discussion_r1840659335